tests/test_derived_meshes.py

```diff
@@ -0,0 +1,187 @@
+import tempfile
+from itertools import product
+from pathlib import Path
+
+import numpy as np
+import pyproj
+import pytest
+import pyvista as pv
+
+from geograypher.cameras.cameras import PhotogrammetryCamera, PhotogrammetryCameraSet
+from geograypher.constants import EARTH_CENTERED_EARTH_FIXED_CRS
+from geograypher.meshes.derived_meshes import (
+    TexturedPhotogrammetryMeshPyTorch3dRendering,
+)
+from geograypher.meshes.meshes import TexturedPhotogrammetryMesh
+
+
+def make_simple_camera():
+    """
+    Create a simple camera looking down at the origin from above, spaced to line up
+    perfectly with the simple mesh.
+    """
+
+    # Focal length in pixels
+    focal = 100
+    # Sensor width/height in pixels
+    sensor = 200
+
+    # Known width of the simple mesh in the X/Y plane
+    mesh_width = 4
+
+    # The equation is just a ratio of triangles
+    # Camera distance (m) = Scene width (m) * focal length (px) / sensor width (px)
+    #
+    #    |.
+    #    |   .               .|
+    # ↑  |       .        .   | ↑
+    # sw |  ←cd→   [cam]  ←f→ | sensor w
+    # ↓  |       .        .   | ↓
+    #    |   .               .|
+    #    |.
+    #
+    cam_to_world = np.array(
+        [
+            [1, 0, 0, 0],
+            [0, -1, 0, 0],
+            [0, 0, -1, mesh_width * focal / sensor],
```

Comment:
nit: maybe describe the rotation matrix is to point the camera in the right direction. I feel like the `-1`s could be mistaken for weird sign convention hacks.